### ISSUE TITLE
Terminal cursor cuts off

### ISSUE THREAD

**Issue by dilanx** [warpdotdev/Warp#2148]
### Discord username (optional)

Dilan#1000

### Describe the bug

The cursor is cut off while a command is running.

### To Reproduce

Run a command that takes time to complete.

### Expected behaviour

Text should be shifted upwards as done in other terminals.

### Screenshots

<img width="1309" alt="Screenshot 2022-11-16 at 7 49 40 PM" src="https://user-images.githubusercontent.com/16329935/202334711-ae95c115-3a51-443c-94b8-78eff279cb24.png">

### Operating System

MacOS

### OS Version

13.0

### Shell Version

zsh 5.8.1

### Warp Version

v0.2022.11.14.14.55.stable_01

### Additional context

Not an urgent issue, just a UI bug that bothers me and doesn't exist with other terminals.

### Does this block you from using Warp daily?

No

**Comment by dilanx** [comment]
@elviskahoro just checked and yes it looks like the issue persists. I added `ZDOTDIR=/` to my `.zshenv` file and as expected it disabled all of my zsh configuration but the cursor is still cut off when running a slow command.

**Comment by nichoj10** [comment]
Hi @dilanx, we're looking into this. I'm not able to repro it yet. It would be super helpful for us if you're willing to share a screen recording of this happening so we can understand the bug better!

**Comment by dilanx** [comment]
Good catch. It looks like if I reset my font to the default "Hack" the second line cursor barely fits but it does fit. I think because it's so close to the bottom, my other font "Comic Code" causes the cutoff. With Monaco, the cursor is basically touching the bottom of the window, so it seems to be different with each font. The font size is 13 and line height is 1.2 (which it appears is default).


https://user-images.githubusercontent.com/16329935/206574303-c6ec5c0d-32a8-41cc-93c3-ae03286a0e22.mov

**Comment by nichoj10** [comment]
Thanks for the video!! It looks like the "Comic Code" font causes some problems with our line height calculation. We'll keep this issue open and will post any updates here.
